test/pspm_test_temp.m

```diff
@@ -0,0 +1,122 @@
+function pspm_test_temp(varargin)
+	% pspm_test_temp is a wrapper script for testing selected testable functions in one
+	% - to be used before any release.
+	%
+	% quit_after_tests : [bool]
+	%                    Define whether the script should quit MATLAB with a success/fail flag
+	%                    after the tests are run.
+	%                    (Default: false)
+
+	%__________________________________________________________________________
+	% PsPM TestEnvironment
+	% (C) 2013 Dominik Bach & Linus Ruettimann (University of Zurich)
+	% Updated Teddy Chao (WCHN, UCL)
+
+	%% Imports
+	import matlab.unittest.TestSuite;
+
+	quit_after_tests = false;
+	if nargin > 0
+		quit_after_tests = varargin{1};
+		assert(islogical(quit_after_tests));
+	end
+
+	%% Build suits
+	suite = [...
+	% TestSuite.fromClass(?pspm_load_data_test), ...
+	% TestSuite.fromClass(?pspm_write_channel_test), ...
+	% TestSuite.fromClass(?pspm_split_sessions_test), ...
+	TestSuite.fromClass(?pspm_trim_test), ...
+	% TestSuite.fromClass(?pspm_find_channel_test), ...
+	% TestSuite.fromClass(?pspm_get_timing_test), ...
+	% TestSuite.fromClass(?pspm_import_test), ...
+	% TestSuite.fromClass(?pspm_prepdata_test), ...
+	% TestSuite.fromClass(?pspm_pp_test), ...
+	% TestSuite.fromClass(?pspm_scr_pp_test), ...
+	% TestSuite.fromClass(?pspm_pulse_convert_test),...
+	% TestSuite.fromClass(?pspm_ren_test), ...
+	% TestSuite.fromClass(?pspm_load1_test), ...
+	% TestSuite.fromClass(?pspm_glm_test), ...
+	% TestSuite.fromClass(?pspm_find_sounds_test), ...
+	% TestSuite.fromClass(?pspm_bf_test), ...
+	% TestSuite.fromClass(?pspm_interpolate_test), ...
+	% TestSuite.fromClass(?pspm_find_valid_fixations_test), ...
+	% TestSuite.fromClass(?pspm_extract_segments_test), ...
+	% TestSuite.fromClass(?pspm_process_illuminance_test), ...
+	% TestSuite.fromClass(?pspm_ecg2hb_test), ...
+	% TestSuite.fromClass(?pspm_convert_unit_test), ...
+	% TestSuite.fromClass(?pspm_align_channels_test), ...
+	% TestSuite.fromClass(?pspm_resp_pp_test), ...
+	% TestSuite.fromClass(?pspm_ecg2hb_amri_test), ...
+	% TestSuite.fromClass(?pspm_path_test), ...
+	% TestSuite.fromClass(?pspm_pupil_correct_eyelink_test), ...
+	% TestSuite.fromClass(?pspm_pupil_correct_test), ...
+	% TestSuite.fromClass(?pspm_pupil_pp_test), ...
+	% TestSuite.fromClass(?set_blinks_saccades_to_nan_test), ...
+	% TestSuite.fromClass(?pspm_blink_saccade_filt_test), ...
+	% TestSuite.fromClass(?pspm_convert_gaze_distance_test), ...
```

Comment:
It is intended to stay commented, or is it a temporary fix?

Reply:
@jbrochar Hi jules, this is just used for testing the target function by me, I need to add it to .gitignore.